### PR TITLE
Implement dashboard page

### PR DESCRIPTION
**Pull request by daniddelrio**
Still lacking dashboard in the navbar

### CODE REVIEW

File: api/main_events/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.0.5 on 2019-02-07 04:43
```
Comment (galacemiguel) [inline api/main_events/migrations/0001_initial.py]: Did you mean to commit this?

File: client/src/components/routes/dashboard/Dashboard.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+import { media } from "style/style-utils";
+
+import axios from "axios";
+
+const DashboardHeading = styled(AppHeading)`
+  font-size: 1.7em;
+  ${media.mdScreen`
+    font-size: 1.7em;
+  `}
+`
```
Comment (galacemiguel) [inline client/src/components/routes/dashboard/Dashboard.js]:
```suggestion
`;
```

File: client/src/components/routes/dashboard/Dashboard.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+const DashboardPendingHeading = styled(DashboardHeading)`
+  ${media.mdScreen`
+    margin-left: 1rem;
+  `}
+`
```
Comment (galacemiguel) [inline client/src/components/routes/dashboard/Dashboard.js]:
```suggestion
`;
```

File: client/src/components/routes/dashboard/Dashboard.js [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    float: right;
+  `}
+`;
+
+const DashboardApprovedCard = styled.div`
+  ${media.mdScreen`
+    margin-top: 1.3rem;
+    margin-right: 1.8rem;
+  `}
+`
```
Comment (galacemiguel) [inline client/src/components/routes/dashboard/Dashboard.js]:
```suggestion
`;
```

File: client/src/components/routes/dashboard/Dashboard.js [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+const DashboardPendingCard = styled.div`
+  ${media.mdScreen`
+    margin-top: 1.3rem;
+    margin-left: 1rem;
+    margin-right: 1.2rem;
+  `}
+`
```
Comment (galacemiguel) [inline client/src/components/routes/dashboard/Dashboard.js]:
```suggestion
`;
```

File: api/main_events/migrations/0001_initial.py (hunk 1 above)
Comment (daniddelrio) [reply api/main_events/migrations/0001_initial.py]: Oops, that shouldn't have been committed

Comment (galacemiguel) [reply api/main_events/migrations/0001_initial.py]: Can you remove it from this PR?

Comment (daniddelrio) [reply api/main_events/migrations/0001_initial.py]: Done

**Pull request merged**